core/slim_sim.cpp

```diff
@@ -8334,38 +8342,23 @@ slim_generation_t SLiMSim::_InstantiateSLiMObjectsFromTables(EidosInterpreter *p
 	if (ret != 0) handle_error("_InstantiateSLiMObjectsFromTables tsk_treeseq_free()", ret);
 	free(ts);
 	
-	// Figure out how many remembered genomes we have; each remembered individual has two remembered genomes
```

Comment:
A lot of deleted code here; can you explain why this code is no longer needed?

Reply:
Yep, of course. I implemented the version in the comment that was previously there: "We could instead simply loop over the nodes and see if the individual they point to is remembered,". I had to to this because the remembered_genome_count no longer counts all the individuals in the tables (as some are retained, not remembered). Once we loop over nodes like this, the checking code in 8361-8365 becomes redundant.

Reply:
OK thanks.

Reply:
>I was surprised not to see any diffs in the simplify algorithm itself; doesn't it need to be changed to respect this retain flag? 

Nope - that's the nice thing. The remembered individuals are marked as samples, and so always get kept (that's how simplify has always worked). The retained individuals are not marked as samples, and so get thrown away automatically by the simplify algorithm. So we only need to expose this functionality to SLiM - hurrah.